thanos/templates/compact-deployment.yaml

```diff
@@ -45,12 +45,11 @@ spec:
         - "compact"
         - "--log.level={{ .Values.compact.logLevel }}"
         - "--objstore.config-file=/etc/config/{{ .Values.objstore.configFile }}"
-        - "--retention.resolution-raw=30d"
-        - "--retention.resolution-5m=120d"
-        - "--retention.resolution-1h=10y"
-        - "--sync-delay=30m"
         - "--data-dir=/var/thanos/store"
         - "--wait"
+        {{- if .Values.compact.extraArgs }}
```

Comment:
Please remove these extraArgs because of #908

Reply:
Thanks, but I had a confusion with the #908 PR. I removed these changes from it - it should be merged here, as this is how it was tested.